src/neighborhood.jl

```diff
@@ -36,6 +38,18 @@ struct Neighborhood
         allowed_mutations = filter(recipe -> !(recipe in frozen_shifts), get_nbhd(shifts))
         new(allowed_mutations, shifts)
     end
+
+    function Neighborhood(shifts::Shifts, frozen_days::Vector{Int}, sample_size::Real)
```

Comment:
A jakby zrobić, że ten konstruktor jest wywoływany przez te wyższe tylko, że z jedynką, byłoby trochę mniej kodu, ma to sens?